console/EAP7-1130_Add_support_to_several_messaging-activemq_resources.adoc

```diff
@@ -0,0 +1,68 @@
+= EAP7-1130 Add support to several messaging-activemq resources
+:author:            Claudio Miranda
+:email:             claudio@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse/
+
+== Overview
+
+Add the following resources to HAL:
+
+  /subsystem=messaging-activemq
+        "connection-factory",
+        "connector",
+        "discovery-group",
+        "external-jms-queue",
+        "external-jms-topic",
+        "http-connector",
+        "in-vm-connector",
+        "pooled-connection-factory",
+        "remote-connector",
+
+Currently there are the same set of resources under /subsystem=messaging-activemq/server=* but they use the embedded ActiveMQ Artemis broker to work. The above mentioned resources uses a remote ActiveMQ Artemis broker and there is no need for the embedded ActiveMQ Artemis server.
+        
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-1130[EAP7-1130]
+
+=== Dev Contacts
+
+* mailto:claudio@redhat.com[Claudio Miranda]
+
+=== QE Contacts
+
+* mailto:pjelinek@redhat.com[Pavel Jelinek]
+
+== Requirements
+
+  /subsystem=messaging-activemq
+        "connection-factory",
+        "connector",
+        "discovery-group",
+        "external-jms-queue",
+        "external-jms-topic",
+        "http-connector",
+        "in-vm-connector",
+        "pooled-connection-factory",
+        "remote-connector",
+
+* Add a menu item in the Messaging Category column named "External ActiveMQ Broker", this is a View to show all resources.
```

Comment:
Could you update this document according to the implementation? To be more specific, there is no "External ActiveMQ Broker"  menu item instead, there is a menu item called "Remote ActiveMQ Server"

Reply:
Done